src/libaktualizr/uptane/directorrepository.cc

```diff
@@ -108,6 +112,12 @@ bool DirectorRepository::updateMeta(INvStorage& storage, Fetcher& fetcher) {
 
     int local_version = rootVersion();
 
+    // if remote_version <= local_version then the director's root metadata are never verified
+    // which leads to two issues
+    // 1. At initial stage (just after provisioning) the root metadata from 1.root.json are not verified
+    // 2. If local_version becomes higher than 1, e.g. 2 than a rollback attack is possible since the business logic
+    // here won't return any error as suggested in #4 of
+    // https://uptane.github.io/uptane-standard/uptane-standard.html#check_root
```

Comment:
Good call. Can you make another tech debt for this? (It's the same logic with the Image repo, so it can all be in one ticket.)

Reply:
Sure.

Reply:
https://saeljira.it.here.com/browse/OTA-4119